client-app/shared/catalog/components/product/properties.vue

```diff
@@ -18,15 +18,29 @@
     >
       <Vendor :vendor="product.vendor" withRating />
     </ProductProperty>
+
+    <a
+      v-if="product.properties && product.properties.length > MAX_DISPLAY_ITEMS"
+      class="flex gap-x-1 mb-4 items-center text-14 text-[color:var(--color-link)] hover:text-[color:var(--color-link-hover)] hover:cursor-pointer underline decoration-dashed"
+      @click="showAll = !showAll"
+    >
+      <VcIcon class="text-[color:var(--color-primary)]" :name="[showAll ? 'chevron-up' : 'chevron-down']" size="xs" />
+      {{
+        showAll == true ? $t("shared.catalog.product_details.see_less") : $t("shared.catalog.product_details.see_more")
```

Comment:
'showAll' has boolean type, it coud be simply?
showAll ? $t("shared.catalog.product_details.see_less") : $t("shared.catalog.product_details.see_more")